client/src/components/ViewProfile.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
+import { Container, Button, Card } from "semantic-ui-react";
+import { getProfile } from "../api/profiles";
+import { useParams } from "react-router-dom";
+
+const ViewProfile = () => {
+  const [profile, setProfile] = useState([]);
+
+  let { profileId } = useParams();
+
+  useEffect(() => {
+    getProfile(profileId).then(response => {
+      setProfile(response);
+      document.title =
+        response.first_name + " " + response.last_name + " Profile";
```

Comment:
```suggestion
        `${response.first_name} ${response.last_name} Profile`;
```
I think most teams now prefer using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for string concatenation so let's do that